.github/workflows/smartbear-issue-opened.yml

```diff
@@ -0,0 +1,56 @@
+name: Issue Opened
+
+on:
+  issues:
+    types:
+      - opened
```

Comment:
Do we want every issue to be opened automatically? I think there should be an explicit action.

Reply:
its one of the options, there is currently 3:
1. commenting on a ticket, or
2. add a label, or
3. whenever creating an issue

So depending on the repo, and usage, we could be creating a ticket for every issue

Reply:
it will be up to the maintainer to choose one based on their preference